game/game.settings.js

```diff
@@ -57,12 +57,7 @@ module.exports = {
 
     treatments: {
         pgg: {
-            description: 'Public Good Game',
-            showBars: true
-        },
-
-        pgg_bars: {
-            description: 'Public good game with results shown with bars',
+            description: 'Klasická verze Public Good Game',
```

Comment:
not sure to merge the Czech description in the main branch. Maybe we can have two branches English/Czech?

Reply:
Yeah, it's mistake... How to withdraw my suggestion?